source/common/http/filter_manager.cc

```diff
@@ -368,6 +368,22 @@ ResponseTrailerMapOptRef ActiveStreamFilterBase::responseTrailers() {
   return parent_.filter_manager_callbacks_.responseTrailers();
 }
 
+void ActiveStreamFilterBase::sendLocalReply(
+    Code code, absl::string_view body,
+    std::function<void(ResponseHeaderMap& headers)> modify_headers,
+    const absl::optional<Grpc::Status::GrpcStatus> grpc_status, absl::string_view details) {
+
```

Comment:
nit: unnecessary whitespace empty lines - the style in Envoy is not to put an empty line as a first line in the body